psi4/src/psi4/libmints/twobody.h

```diff
@@ -203,6 +206,8 @@ class PSI_API TwoBodyAOInt {
     /*
      * Sieve information
      */
+    /// Is sieve initialized?
+    bool initialized() { return initialized_; };
```

Comment:
This is unintuitive that it is referring to the sieve. If I see initialized() I would guess it refers to the object not a subcomponent of the object.